### PR TITLE
Service manual

### PR DESCRIPTION
**Pull request by rossjones**
Adds a temporary static page showing potential service manual content.

### CODE REVIEW

File: app/views/pages/service_manual.html.erb [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+          This KPI shows why calls are made and gives an indication of how the number of calls can be reduced.
+        </p>
+        <p>
+        Reasons include:</p>
+        <ul class="list-bullet">
+          <li>to perform a transaction</li>
+          <li>to get information</li>
+          <li>to chase progress</li>
+          <li>to challenge a decision</li>
+          <li>Other</li>
```
Comment (olly) [inline app/views/pages/service_manual.html.erb]: Should this be lowercase, to be consistent with the rest of this list and the previous breakdown?

Comment (rossjones) [reply app/views/pages/service_manual.html.erb]: Oops. Will fix.

**Comment by rossjones** [comment]
I've asked LW where they should go, I'd expect at least one of them should be an email address.

**Comment by rossjones** [comment]
Links added, should be good to go.

**Comment by olly** [comment]
LGTM 👍

**Pull request merged**